pyproject.toml

```diff
@@ -55,3 +54,24 @@ build-backend = "flit_core.buildapi"
 
 [tool.flit.sdist]
 include = [".sqlfluff"]
+
+[tool.ruff]
+target-version = "py310"
+
+lint.select = [
+    "A",  # prevent using keywords that clobber python builtins
+    "B",  # bugbear: security warnings
+    "E",  # pycodestyle
+    "F",  # pyflakes
+    "I",  # isort
+    "ISC",  # implicit string concatenation
+    "PLE",  # pylint errors
+    "RUF",  # the ruff developer's own rules
+    "UP",  # alert you when better syntax is available in your python version
+]
+lint.ignore = [
+    "ISC001"  # Recommended ingore from `ruff format`
+]
+
+[tool.ruff.lint.per-file-ignores]
+"cumulus_library/schema/valueset.py" = ["E501"]
```

Comment:
Why is github highlighting this stuff red?

(Also, maybe a comment about why this ignore / when if ever it could be removed)

Reply:
will add, lemme check the formatting, might need to move the x.y into [tool.ruff.x] y or something

Reply:
fixed formatting, added comment.